projects/admin/src/app/record/detail-view/document-detail-view/document-detail-view.component.ts

```diff
@@ -58,6 +58,8 @@ export class DocumentDetailViewComponent implements DetailRecord, OnInit, OnDest
   /** Css classes for dd in template */
   ddCssClass = 'col-sm-6 col-md-8 mb-0';
 
+  import: boolean = false;
```

Comment:
Missing docstring. Maybe the param could be more explicit (like 'import_route' or something similar)